client/src/main/java/tw/waterball/survive/io/client/view/StandardGameView.java

```diff
@@ -1,25 +1,46 @@
 package tw.waterball.survive.io.client.view;
 
-
+import tw.waterball.survive.io.client.controller.PlayerController;
 import tw.waterball.survive.io.game.sprite.Sprite;
 import tw.waterball.survive.io.game.sprite.role.Role;
 import tw.waterball.survive.io.game.sprite.weapon.Weapon;
 
+import javax.swing.*;
+import java.awt.*;
+import java.util.Collections;
 import java.util.List;
 
 /**
- * @author - c11037at@gmail.com
+ * @author - timt1028@gmail.com
  */
 public class StandardGameView extends GameView {
+    private PlayerController playerController;
+    private Canvas canvasGame = new Canvas();
+
+    public StandardGameView(PlayerController playerController) {
+        this.playerController = playerController;
+        setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+        setLocation(300, 300);
+        setSize(600, 600);
+        setVisible(true);
+        setContentPane(canvasGame);
+    }
+
+    public static void main(String[] args) {
+        StandardGameView standardGameView = new StandardGameView(new PlayerController());
+        standardGameView.launch();
+    }
 
     @Override
     public void launch() {
+    }
 
+    private void paintSprite(Graphics g, Sprite sprite) {
```

Comment:
put this method below `Canvas` since `Canvas` uses it.

The invokee should be put below the invoker.